src/schema/nmdc.yaml

```diff
@@ -791,15 +791,18 @@ slots:
 
   eluent_introduction_category:
     range: EluentIntroductionCategoryEnum
-    description: how a processed sample is introduced into a mass spectrometer, for example liquid chromatography or direct infusion through a syringe
+    description: Details how a processed sample is introduced into a mass spectrometer
```

Comment:
Can we modify this description to remove the word "Details"; this slot doesn't give details. 
 Maybe `A high-level description for how the processed sample is introduced into a mass spectrometer.`  Or something more elegant?

Reply:
changed in most recent commit